internal/sdkv2provider/schema_cloudflare_queue.go

```diff
@@ -17,5 +17,68 @@ func resourceCloudflareQueueSchema() map[string]*schema.Schema {
 			Required:    true,
 			Description: "The name of the queue.",
 		},
+		"consumer": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "Array of consumers.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"created_on": {
+						Description: "Created on date",
+						Type:        schema.TypeString,
+						Optional:    true,
+						ForceNew:    true,
```

Comment:
you don't want `ForceNew` everywhere otherwise you'll constantly be recreating the resource despite not having changes.

Reply:
ok, removing them